virttest/utils_conn.py

```diff
@@ -774,19 +773,23 @@ def conn_setup(self):
         # require a list data type
         sasl_allowed_users = self.sasl_allowed_users
         listen_addr = self.listen_addr
-
-        # edit the /etc/sysconfig/libvirtd to add --listen args in libvirtd
-        pattern_to_repl = {r".*LIBVIRTD_ARGS\s*=\s*\"\s*--listen\s*\".*":
-                           "LIBVIRTD_ARGS=\"--listen\""}
-        self.remote_syslibvirtd.sub_else_add(pattern_to_repl)
-
-        # edit the /etc/libvirt/libvirtd.conf
-        # listen_tcp=1, tcp_port=$tcp_port, auth_tcp="none"
-        # listen_tcp=1, tcp_port=$tcp_port, auth_tcp=$auth_tcp
-        pattern_to_repl = {r".*listen_tls\s*=.*": 'listen_tls=0',
-                           r".*listen_tcp\s*=.*": 'listen_tcp=1',
-                           r".*tcp_port\s*=.*": 'tcp_port="%s"' % (tcp_port),
-                           r".*auth_tcp\s*=.*": 'auth_tcp="%s"' % (auth_tcp)}
+        pattern_to_repl = {}
+        if not libvirt_version.version_compare(5, 6, 0, server_session):
+            # edit the /etc/sysconfig/libvirtd to add --listen args in libvirtd
+            pattern_to_repl = {r".*LIBVIRTD_ARGS\s*=\s*\"\s*--listen\s*\".*":
+                               "LIBVIRTD_ARGS=\"--listen\""}
+            self.remote_syslibvirtd.sub_else_add(pattern_to_repl)
+
+            # edit the /etc/libvirt/libvirtd.conf
```

Comment:
I am a little concerning how this will be compatible with near coming feature :split daemons, where /etc/libvirt/libvirtd.conf doesn't take effect.

Reply:
@chunfuwen , the modular daemon is another story. I would like to fix the original error in this PR. If need, I can file another PR to solve modular daemon issue. I do not prefer to messing them up. Thanks.